### PR TITLE
QS-298 | Fix Logout Guidance - Swift

### PR DESCRIPTION
**Pull request by benlizar**

### CODE REVIEW

File: articles/quickstart/native/ios-swift/_includes/_login_centralized.md [hunk 1]
```diff
@@ -103,3 +103,29 @@ After the user authenticates, their information is returned in a `credentials` o
...
+To clear the session on the server side you need to invoke the `clearSession` method. Add the following snippet:
+
+```swift
+// HomeViewController.swift
+@swift
+Auth0
+    .webAuth()
+    .clearSession(federated:false){
+        switch $0{
+        case true:
```
Comment (lbalmaceda) [inline articles/quickstart/native/ios-swift/_includes/_login_centralized.md]: Can you correct the indentation of this switch, please?

File: articles/quickstart/native/ios-swift/_includes/_login_centralized.md [hunk 2]
```diff
@@ -103,3 +103,29 @@ After the user authenticates, their information is returned in a `credentials` o
...
+            ...
+        case false:
+            ...
+        }
+    }
+```
+Go to your [Dashboard Settings](${manage_url}/#/applications/${account.clientId}/settings) and make sure that the **Allowed Logout URL** field contains the following logout callback URL:
+
+```text
+{PRODUCT_BUNDLE_IDENTIFIER}://${account.namespace}/ios/{PRODUCT_BUNDLE_IDENTIFIER}/callback
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/_includes/_login_centralized.md:129]: @benlizar I suggest adding a note after this snippet reminding the user that they have to replace `{PRODUCT_BUNDLE_IDENTIFIER}` with the real bundle identifier value from their app.

File: articles/quickstart/native/ios-swift/download.md [hunk 3]
```diff
@@ -2,8 +2,12 @@ To run the sample follow these steps:
 
 1) Set the **Callback URL** in the [Application Settings](${manage_url}/#/applications/${account.clientId}/settings) to
 ```text
-auth0.samples.Auth0Sample://${account.namespace}/ios/auth0.samples.Auth0Sample/callback
+{PRODUCT_BUNDLE_IDENTIFIER}://${account.namespace}/ios/{PRODUCT_BUNDLE_IDENTIFIER}/callback
```
Comment (stevehobbsdev) [inline articles/quickstart/native/ios-swift/download.md:5]: Similar comment as above.

File: articles/quickstart/native/ios-swift/_includes/_login_centralized.md (hunk 2 above)
Comment (lbalmaceda) [reply articles/quickstart/native/ios-swift/_includes/_login_centralized.md:129]: I was going to suggest this as well. Android does it.

image

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**